openedx/features/job_board/urls.py

```diff
@@ -1,9 +1,9 @@
 from django.conf.urls import url
 
-from .views import list_jobs, show_job_detail
+from .views import JobsListView, show_job_detail
 
 
 urlpatterns = [
-    url(r'^jobs/list/$', list_jobs, name='job_list'),
-    url(r'^jobs/list/[0-9]$', list_jobs, name='job_detail'),
+    url(r'^jobs/list/$', JobsListView.as_view(), name='job_list'),
+    url(r'^jobs/list/[0-9]*$', show_job_detail, name='job_detail'),
```

Comment:
Please update * with +. As * means zero or more and + means one or more repetition.

Reply:
Done. Thanks for pointing this out!